run.py

```diff
@@ -696,6 +729,27 @@ def add_tag_wrapper(container, tag: str) -> None:
     container.add_tag(tag)
 
 
+def long_redcap_interval_tag(
+    session: SessionListOutput, hdr_fields: dict, record: dict
+) -> None:
+    """Checks whether the interval between the session and redcap record is > 2 weeks.
+    If so, tags the session with 'long-redcap-interval_unsent'."""
+    max_delta = timedelta(days=14)
+    interval_delta = (
+        datetime.strptime(record['consent_timestamp'], DATETIME_FORMAT_RC)
+        - hdr_fields['date']
+    )
+    if interval_delta > max_delta or interval_delta < -max_delta:
```

Comment:
can simplify with just the absolute value
```suggestion
    interval_delta = abs(
        datetime.strptime(record['consent_timestamp'], DATETIME_FORMAT_RC)
        - hdr_fields['date']
    )
    if interval_delta > max_delta:
```